code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -182,3 +182,7 @@
 
 /mob/living/simple_animal/drone/cogscarab/ratvar_act()
 	fully_heal(TRUE)
+
+/obj/item/drone_shell/dusty
+	name = "dust-caked drone shell"
```

Comment:
Please rename this. I keep confusing it with "dust-caked golem shell". I am excited about golems, but not drones.